advanced/Templates/pihole-FTL.service.ini

```
@@ -0,0 +1,37 @@
+[Unit]
+Description=pihole-FTL Domain Name Server
+Documentation=man:pihole-FTL(8)
+After=remote-fs.target
+
+# Alternative based on https://wiki.debian.org/Hardening/Daemons#bind
+#   allows other services which rely on DNS to use pihole-FTL, but means
+#   breaking if config relies on remote-fs
+# After=network.target
+# Wants=nss-lookup.target
+# Before=nss-lookup.target
+
+After=resolvconf.service
+Wants=resolvconf.service
+# fail since dnsmasq is embedded
+Conflicts=dnsmasq.service
+
+[Service]
+# Run as unprivileged user, use AmbientCapabilities to assign capabilities
+User=pihole
+Group=pihole
+
```

Shouldn't we add
```suggestion
Type=forking
PIDFile=/var/run/pihole-FTL.pid
```
?

---

No we're running pihole-FTL in no-daemon mode, since FTL's double-fork daemon implementation actually seems to cause issues. So the implied `Type=simple` is used where systemd maintains direct control over the process.